lib/ansible/modules/cloud/amazon/ec2_vpc_dhcp_option.py

```diff
@@ -376,6 +376,23 @@ def main():
                 new_options['ntp-servers'],
                 new_options['netbios-name-servers'],
                 new_options['netbios-node-type'])
+
+            # wait for dhcp option to be accessible
+            found_dhcp_opt = False
+            start_time = time()
+            while time() < start_time + 300:
```

Comment:
This all seems a bit boilerplate - can this not be done with retries=100, delay=3, backoff=1 ?

Reply:
As noted in #ansible-aws, this is a boto module so to use AWSRetry would require porting. Going to hold off on porting for now.

Reply:
Yeah, of course, I hadn't spotted that it was boto.